api.md

```diff
@@ -928,23 +928,30 @@ it is not possible to chain multiple commands together in one call.
 |Name|Description|
 |---|---|
 | `node_list` | An array of data nodes where the command should be executed. Defaults to all data nodes if not specified. |
+| `transactional` | Allows to specify if the execution of the statement should be transactional or not. Defaults to TRUE. |
 
 #### Sample Usage [](distributed_exec-examples)
 
 Create the role `davide` across all data nodes in a distributed database:
 
 ```sql
-SELECT distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
+CALL distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
```

Comment:
Maybe use some kind of password template here instead? Feels like some users might create roles using this password :)

Reply:
Thank you for noting. I switched the example to use the one from the certificate authorisation setup.

Reply:
One more thing. I think this example should be in sync with @WireBaron update on authentication setup, regarding required usage of `LOGIN`, e.g.:

`CALL distributed_exec($$ CREATE USER testrole WITH LOGIN PASSWORD 'password' $$); `

Reply:
It might be also a good idea to put a link to that document

Reply:
@pmwkaa I use the example from the certification authorisation. No password is needed in such case.